.tests/supavisor-logs/config.yaml

```diff
@@ -0,0 +1,10 @@
+parsers:
+  - crowdsecurity/dateparse-enrich
+  - ./parsers/s01-parse/crowdsecurity/supavisor-logs.yaml
```

Comment:
you are missing the base s00 parser `crowdsecurity/syslogs-logs` please add and regenerate tests